### ISSUE TITLE
Not Working properly

### ISSUE THREAD

**Issue by CyclonePT** [sadboilogan/discordlyrics-v2#1]
Hey!
I just used your first version of this project and I must say that it is amazing!
I tried this version, but it doesn't seem to be working, with errors like config.default needs to be config.json to run the code.
Are you still working on this project/ do you plan to update it soon?
Compliments,
Cyclone

**Comment by logan-mcgee** [comment]
uh, if you copy the config from the first version it *should* work fine, kinda forgot about adding that mechanic into this

**Comment by CyclonePT** [comment]
I haven't tried yet, but do you have a public server or would you be willing to add me on discord to talk more freely? I just think this is incredible and would like to chat with you about it

**Comment by logan-mcgee** [comment]
no, I don't have any real way of getting in contact simply as many times public support just doesn't seem to work out as you hope, copying over the config should work though, as I had the same thing a couple of weeks back and I solved it the same way

**Comment by CyclonePT** [comment]
Do you know how to fix the error "refreshToken ERR: Error: Request failed with status code 400" when doing "node index.js"?

**Comment by liechtenberg** [comment]
I have the same^

**Comment by vz76** [comment]
> Do you know how to fix the error "refreshToken ERR: Error: Request failed with status code 400" when doing "node index.js"?

same issue as above! copied over config.json and token.json from v1 but I get this error.
